docs/content/en/docs/troubleshooting/_index.md

```diff
@@ -32,6 +32,56 @@ environment for debugging. The tool is meant to be used for debugging a single
 Tetragon agent node. Note that in the context of Kubernetes, the command needs
 to be run from inside the Tetragon Pod's container.
 
+Key Information Collected by the Bugtool:
+
+- Tetragon configuration
+- Network configuration
+- Kernel configuration
+- eBPF maps
+- Process traces (if tracing is enabled)
+
+## Running tetra bugtool Demonstrations
+
+### Kubernetes
+
+1. Identify Tetragon Pod:
+
+   ```bash
+   kubectl get pods -n <tetragron-namespace> -l app.kubernetes.io name=tetragon
+   ```
+
+2. Execute tetra bugtool within the Pod:
+
+   ```bash
+   kubectl exec -n <tetragron-namespace> <tetragron-pod-name> -- tetra bugtool
+   ```
+
+### Container Installation
+
+1. Enter the Tetragon Container:
+
+   ```bash
+   docker exec -it <tetragron-container-id>
+   ```
+
+2. Run tetra bugtool:
+
+   ```bash
+   tetra bugtool
+   ```
```

Comment:
Can you replace this with the step to retrieve the created archive from the container fs to your local system (using docker cp for example)